lib/compressors/html-minifier.js

```diff
@@ -51,7 +50,7 @@ module.exports = compressHTMLMinifier;
  */
 
 function compressHTMLMinifier(settings, content, callback, index) {
-  var options = extend(defaultOptions, settings.options);
+  var options = Object.assign(defaultOptions, settings.options);
```

Comment:
Sure you didn't forgot an initial `{}` in Object.assign? you are mutating defaultOptions now

Reply:
Yes, indeed